scripts/battlefields/Apollyon/se_apollyon.lua

```diff
@@ -123,7 +137,7 @@ area.groups =
             [xi.mod.HTH_SDT] = 0,
             [xi.mod.SLASH_SDT] = 1250,
         },
-        death = function(mob, count)
+        death = function(battlefield, mob, count)
```

Comment:
General comment, and not a requirement, but would like to see a newline after any multiline-block table definition

Reply:
Can be easily done